app/services/content-service/logging.js

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
It's a bit strange having a different "logging" file in each part of the app. Looks like most of this code can be reused. Why not do that?